esx_service/utils/auth.py

```diff
@@ -34,6 +34,7 @@
 
 SIZE = 'size'
 DEFAULT_TENANT = '_DEFAULT'
+DEFAULT_TENANT_UUID = '_DEFAULT_TENANT_UUID'
```

Comment:
Personally I feel using some special string that can be easily identified (so as to differ from the auto-generated uuid string) will be helpful for debugging.